packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -1843,73 +1642,66 @@ describe( 'Addition Settings UI', () => {
 		);
 
 		// Assert the default "checked" states match the expected.
-		expect( settingControlsInputs[ 0 ].checked ).toEqual( false );
-		expect( settingControlsInputs[ 1 ].checked ).toEqual( true );
+		expect( settingControlsInputs[ 0 ] ).not.toBeChecked();
+		expect( settingControlsInputs[ 1 ] ).toBeChecked();
```

Comment:
Looks like the querying in this test can be a lot simplified by doing something like `screen.getByRole( 'checkbox', { name: 'my label' } )` instead of finding the `label`s first and going backwards.

Reply:
Good call, addressed in b31e97f3000b13236467e24898ebbe30fae28274